README.rst

```diff
@@ -1159,10 +1159,10 @@ Choose from the list of available rules:
 
 * **php_unit_set_up_tear_down_visibility**
 
-  Changes the visibility of the setUp and tearDown functions of phpunit to
-  protected, to match the PHPUnit TestCase.
+  Changes the visibility of the ``setUp()`` and ``tearDown()`` functions of
+  phpunit to ``protected``, to match the PHPUnit TestCase.
 
-  *Risky rule: this fixer may change functions named setUp or tearDown outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
+  *Risky rule: this fixer may change functions named ``setUp()`` or ``tearDown()`` outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
```

Comment:
Ha, not sure about this:

![screen shot 2018-03-20 at 22 43 09](https://user-images.githubusercontent.com/605483/37684383-23bea7de-2c90-11e8-9c0b-99e31139da1f.png)

Reply:
really like the change, sad that the render is not good :(

Reply:
Yep, that's sad :/ I think the change here should be reverted.

Reply:
looking at it again, this is already the case for more risky descriptions and I prefer to have it the same way everywhere (and maybe changed everywhere in another PR) than having to forms.